packages/augur-ui/src/modules/modal/modal.styles.less

```diff
@@ -1701,6 +1701,70 @@ div.ButtonsRow:last-of-type {
         }
       }
     }
+
+    > div:nth-of-type(2) {
+      display: grid;
+      grid-template-columns: 1fr 1fr 1fr;
+      grid-template-rows: auto;
+      grid-gap: @size-16;
+      align-items: flex-end;
+      justify-content: space-between;
+      background-color: #161518;
+      border-radius: @size-3;
+      padding: @size-12;
+
+      > div {
+        width: 100%;
+      }
+
+      > div:first-of-type {
+        label {
+          .mono-9-bold;
+
+          color: #AFA7C1;
+        }
+
+        input[type='number'] {
+          .mono-11;
+
+          border: @size-1 solid #3C3B43;
+          color: #E9E5F2;
+          padding: @size-6 @size-16;
+        }
+      }
+
+      > div:nth-of-type(2) {
+        margin: 0 @size-5;
+        border-right: @size-1 solid #3C3B43;
+
+        > div {
+          > span:first-of-type {
+            .mono-10-bold;
+
+            color: #E9E5F2;
+          }
+
+          > span:last-of-type {
+            .text-10-bold;
+
+            color: #848294;
```

Comment:
colors should not be hardcoded. should be using constants

Reply:
Yeah, I was leaving those for the polish PR. As soon as I get the layout done I’ll check and change those, most likely on the next PR, if that’s ok.

Reply:
not going to allow hardcode colors to be merged up when we need to get the color constants worked out. There needs to be a really good reason to merge up hardcoded colors and I don't think this is a good reason.

Reply:
Understood, won't happen again. Fixed on the latest commit (https://github.com/AugurProject/augur/pull/4501/commits/8ec12168e2ad43cd090f8ac47ba17dad136e3ca4).

Reply:
main reason is so much effort has gone into consolidating and coordinating the color pallet.